src/prometheus_alert_rules/KubeflowMinioServices.rules

```diff
@@ -23,131 +23,61 @@ groups:
         {{ $labels.juju_charm }} unit {{ $labels.juju_model }}/{{ $labels.juju_unit }} has been unreachable at least 50% of the time over the last 10 minutes.
         LABELS = {{ $labels }}
 
-- name: MinioDiskAlerts
-  rules:
-  - alert: MinIO_Low_Disk_Space
-    expr: minio_node_disk_free_bytes < 10 * 1024 * 1024 * 1024
-    for: 5m
-    labels:
-      severity: critical
-    annotations:
-      summary: "Low disk space on MinIO node"
-      description: |
-        MinIO node has less than 10GB of free disk space.
-        LABELS = {{ $labels }}
-
-  - alert: MinIO_High_Disk_Usage
-    expr: (minio_node_disk_used_bytes / minio_node_disk_total_bytes) > 0.9
+  # Disk space
+  - alert: Disk_Space_Filling_Up
+    expr: predict_linear(minio_node_disk_used_bytes[1h], 86400) >= minio_node_disk_total_bytes
     for: 5m
     labels:
       severity: high
     annotations:
-      summary: "High disk usage on MinIO node"
-      description: |
-        MinIO node has over 90% of its disk space used.
-        LABELS = {{ $labels }}
+      description: "MinIO will fill all available space in less than 1 day."
 
-- name: MinioS3RequestAlerts
-  rules:
-  - alert: MinIO_S3_Request_Errors
-    expr: rate(minio_s3_requests_errors_total[2m]) / rate(minio_s3_requests_total[2m]) > 0.1
-    for: 5m
-    labels:
-      severity: high
-    annotations:
-      summary: "High rate of S3 request errors"
-      description: |
-        More than 10% of S3 requests are resulting in errors.
-        LABELS = {{ $labels }}
-
-  - alert: MinIO_S3_Request_Auth_Failures
-    expr: rate(minio_s3_requests_rejected_auth_total[2m]) > 1
+  - alert: MinIO_Low_Disk_Space
+    expr: minio_node_disk_free_bytes < 1 * 1024 * 1024 * 1024
```

Comment:
suggestion: use free disk percentage rather than the 1GB limit
we can use `minio_node_disk_total_bytes` to calculate the percentage

Reply:
So for this one we discussed with Leon that it's better to have a specific number as bytes, rather than a specific number as percentage.

The reason here could be that the PVC could be 10TB, so with a percentage of 1% (example) then this would be 100Gb.

Ideally I was thinking of putting this to something quite small, to denote that it run out of space but was afraid of how the code might behave in a disk congestion situation. Maybe set this to something like 256MB?

Reply:
I see your point, I think we can keep it as 1GB and adjust based on feedback if necessary.